packages/server/graphql/intranetSchema/mutations/enableSAMLForDomain.ts

```diff
@@ -0,0 +1,38 @@
+import {GraphQLNonNull, GraphQLString, GraphQLBoolean} from 'graphql'
+import getRethink from '../../../database/rethinkDriver'
+
+const enableSAMLForDomain = {
+  type: new GraphQLNonNull(GraphQLBoolean),
+  description: 'Enable SAML for domain',
+  args: {
+    url: {
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    domain: {
+      type: GraphQLNonNull(GraphQLString),
+    },
+    metadata: {
+      type: GraphQLNonNull(GraphQLString),
+    },
+    cert: {
```

Comment:
`cert` can be removed (the X.509 cert is stored within the metadata & is not required on its own)